src/status_im/wallet/collectibles/core.cljs

```diff
@@ -54,7 +54,7 @@
 (handlers/register-handler-fx
  :load-kitties
  (fn [{db :db} [_ ids]]
-   {:db db
+   {:db (update-in db [:collectibles] merge {ck (sorted-map-by >)})
```

Comment:
could you elaborate on why we need this? thanks

Reply:
The original bug report mentioned that the displayed order of the cats is different than on the website. That's because they are unsorted.

If you mean why that code in that precise spot, it's because you must have the sorted map there first, before you add any cats, if you want to maintain a sorted order after the merges. Putting it there seemed like the least obtrusive way to do it. (I don't have to touch the initialization, and every other function stays the same)

Did you try it and compare?